.gitignore

```diff
@@ -2,3 +2,4 @@
 !/.github/
 /vendor/
 /composer.lock
+/**/.phpunit.result.cache
```

Comment:
I believe the `/**/` prefix is meaningless. That is, `.phpunit.result.cache` means the same thing. However, I think we can just generalize this to `*.cache`.